cmd/snap-confine/spread-tests/main/network-namespace-sharing/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: The /run/netns directory propagates events outwards
```

Comment:
This looks good but those spread tests don't run (yet). Could you please add this test under `tests/main` instead?